doc/Table.md

```diff
@@ -0,0 +1,281 @@
+| ition        | int(11) | Transition energy switch 
+| CHRO         | text    | /  
+| TUNE         | text    | / 
+| POST         | text    | 'POST' 
+| POS1         | text    | ''
+| ndafi        | int(11) | Number of data files to be processed. 
+| tunex        | double  | Horizontal tune 
+| tuney        | double  | Vertical tune 
+| inttunex     | double  | Values of the horizontal tune (integer part) to be added to the averaged phase advance and to the Q values of the FFT analysis
+| inttuney     | double  | Values of the vertical tune... 
+| DIFF         | text    |'/DIFF' 
+| DIF1         | text    |/ 
+| pmass        | double  | Proton mass 
+| emit_beam    | double  | Emittance 
+| e0           | int(11) | Energy 
+| bunch_charge | double  | Bunch charge 
+| CHROM        | int(11) | To correct for slight differences between MAD and SixTrack the chromaticity is routinely corrected by setting 'chrom' to 1 and using chrom_eps=0.0000001 
+| chrom_eps    | double  | chrome_eps 
+| dp1          | double  | Momentum deviation for particle 1 
+| dp2          | double  | Momentum deviation for particle 2 
+| chromx       | int(11) | chromx 
+| chromy       | int(11) | chromy 
+| rfvol        | double  | RF Voltage 
+| sigz         | double  | r.m.s bunch length 
+| sige         | double  | r.m.s energy spread 
+
+## oneturn_sixtrack_result
+| Field     | Type       | Description 
+|-----------|------------|------
+| task_id   | int(11)    | The task id 
+| wu_id     | int(11)    | The work unit id 
+| betax     | float      | Horizontal beta-function
+| betax2    | float      | Secondary horizontal beta-function 
+| betay     | float      | Vertical beta-function 
+| betay2    | float      | Secondary beta-function 
+| tunex     | float      | Horizontal tune 
+| tuney     | float      | Vertical tune 
+| chromx    | float      | Horizontal chromaticity 
+| chromy    | float      | Vertical chromaticity 
+| x         | float      | x 
+| xp        | float      | xp
+| y         | float      | y 
+| yp        | float      | yp 
+| z         | float      | z 
+| zp        | float      | zp 
+| chromx_s  | float      | chromx obtained by calculation (second\_turn\_tune-first\_turn\_tune)/chrom_eps
+| chromy_s  | float      | chromy obtained by calculation 
+| chrom_eps | float      | Used for calculating chromaticity 
+| tunex1    | float      | First turn horizontal tune 
+| tuney1    | float      | First turn vertical tune 
+| tunex2    | float      | Second turn horizontal tune 
+| tuney2    | float      | Second turn vertical tune 
+| mtime     | bigint(20) | Last modification time 
+
+## sixtrack_wu
+| Field         | Type       | Description 
+|---------------|------------|------
+| wu_id         | int(11)    | The work unit id 
+| preprocess_id | int(11)    | The work unit id of preprocess job 
+| job_name      | text       | The job name
+| input_file    | blob       | The input file needed by the sixtrack.py 
```

Comment:
the description is a bit obscure - maybe a newcomer does not (yet) know what sixtrack.py is/does...